### PR TITLE
Add Microsoft Graph PowerShell option

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/10039

### CODE REVIEW

File: Teams/includes/mtr-device-config-password-include.md [hunk 1]
```diff
@@ -10,6 +10,20 @@ To avoid having to reset the resource account's password and then logging into e
 
 Follow the steps in one of the following tabs to turn off password expiration:
 
+#### [**Microsoft Graph PowerShell**](#tab/graph-powershell-password/)
+
+First, Connect to Graph PowerShell:
```
Comment (JohanFreelancer9) [inline Teams/includes/mtr-device-config-password-include.md:15]:
```suggestion
First, connect to Graph PowerShell:
```

File: Teams/includes/mtr-device-config-password-include.md [hunk 2]
```diff
@@ -10,6 +10,20 @@ To avoid having to reset the resource account's password and then logging into e
...
+
+This example sets the password for the account ConferenceRoom01@contoso.com to never expire.
+
+```PowerShell
+Update-MgUser -UserId ConferenceRoom01@contoso.com -PasswordPolicies DisablePasswordExpiration -PassThru
+```
+
 #### [**Azure Active Directory 2.0**](#tab/azure-active-directory2-password/)
 
 First, Connect to Active Directory PowerShell:
```
Comment (JohanFreelancer9) [inline Teams/includes/mtr-device-config-password-include.md:29]:
```suggestion
First, connect to Active Directory PowerShell:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
Hi @dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**